server/odc-service/src/main/java/com/oceanbase/odc/service/session/model/SqlAsyncExecuteReq.java

```diff
@@ -54,4 +55,8 @@ public boolean ifAddROWID() {
         }
         return this.addROWID;
     }
+
+    public boolean ifEditPLSql() {
```

Comment:
this class is designed for sql execution, don't add any non-related logic here, please

Reply:
But the implementation of ob oracle's pl editing functionality is to call the `streamExecute` method, and I think ob mysql should do the same